app/controllers/repairs_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RepairsController < ApplicationController
+    def index
+        @repair = Repair.all 
+    end
+    def new
```

Comment:
To follow the style guidelines, add a blank line to separate all of the function definitions in this file.